doc/reference/annotations.rst

```diff
@@ -388,6 +388,8 @@ Available Types:
 |                                                          | into Doctrine's ArrayCollection class.           |
 +----------------------------------------------------------+--------------------------------------------------+
 
+*) If the standalone jms/serializer is used then default format is `\DateTime::ISO8601` (which is not compatible with ISO-8601 despite the name). For jms/serializer-bundle the default format is `\DateTime::ATOM` (the real ISO-8601 format) but it can be changed in [configuration](https://jmsyst.com/bundles/JMSSerializerBundle/master/configuration#configuration-block-2-0).

```

Comment:
I'm not sure how to make the link correctly. This is probably not what you want here.

Reply:
> (which is not compatible with ISO-8601 despite the name).

can you elaborate? why is not compatible?

Reply:
See PHP documentation: http://php.net/manual/en/class.datetime.php#datetime.constants.atom

```
DateTime::ISO8601
DATE_ISO8601
    ISO-8601 (example: 2005-08-15T15:52:01+0000)

        Note: This format is not compatible with ISO-8601, but is left this way for backward compatibility reasons. Use DateTime::ATOM or DATE_ATOM for compatibility with ISO-8601 instead. 
```

Reply:
ok!

Reply:
The difference between `DateTime::ISO8601` and `DateTime::ATOM` is the `:` in timezone. According to ISO-8601 it should be there. PHP does not have it there because of a mistake when the constant was added. It probably won't be fixed because of BC.